src/index.scss

```diff
@@ -1,34 +1,27 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900;1000&display=swap');
 
-/*
-if there's any unexpected styling (esp. coloring), check node_modules\bootstrap\dist\bootstrap.css
-(you'll have to restart the app for changes to show)
-*/
-
-:root{
-  --dullblue: #465491;
-
-  /*for hover transparency transitions*/
-  --dullblueRGBA1: #46549100;
-  --dullblueRGBA2: #465491FF;
-
-  --whiteRGBA1: #FFFFFF00;
-  --whiteRGBA2: #FFFFFFFF;
-}
+$blue: #465491;
+$white: #FFFFFF;
```

Comment:
You could also just use CSS' `white` :p

Reply:
Hmm, I prefer consistency
And what if design randomly decides to make the site a slightly darker shade of white!

Reply:
If design does that that's a quick refactor -- there's no plans for that currently, and we'd have to rename the variable anyways

Having `$white` _and_ `white` creates confusing inconsistency, as, well, only `#ffffff` is white.

If you really want to, I'd suggest`$background`, which is much more semantic and will save more refactoring later.